\chapter{Impact of \textit{de novo} Splicing Mutations on Human Brain Spliceoform Dynamics in Autism Spectrum Disorder}
\section{Background}
Alternative splicing is the process by which the exons of a single gene may be differentially included or excluded in distinct mRNA transcripts to give rise to a plurality of distinct products. It is well known that much of the diversity in eukaryotic biology can be attributed to this alternative splicing of mRNA transcripts, and studies of high-throughput sequencing have shown 95-100\% of human pre-mRNAs encompassing more than a single exon are processed to yield multiple mature mRNAs \cite{Pan2008, Wang2008}. The differential regulation and production of alternatively spliced mRNA transcripts can also be dictated by spatial or temporal cues, such as tissue specificity or developmental periods, respectively \cite{Nilsen2010, Porter2018}, with the brain exhibiting higher numbers of alternative splicing events relative to other tissues\cite{Pan2008, Yeo2004, Xu2002}.\par

While the specific functions of individual alternatively spliced isoforms are largely unexplored, these may be inferred through the use of isoform co-expression networks under the idea that, although co-regulation of a pair of isoforms may not necessarily imply that they are related, large sets of isoforms that are co-expressed in a similar manner are likely to be enriched in a central function \cite{Carter2004, Stuart2003}. 

\section{Results}

\subsection{Specific co-expression modules are highly impacted by \textit{de novo} ASD SNVs}

We performed isoform co-expression network analysis using the \textit{WGCNA} R package, which resulted in 50 modules of highly co-expressed isoforms. 

In order to assess the impact of \textit{de novo} ASD splice-site mutations and \textit{de novo} LoF mutations on these co-expression modules, we first scored each module by their splicing mutation rates. We identify several relatively high scoring modules in the cases of either splice-site or total LoF mutations derived from the case cohort. In the case of LoF case mutation rate, M39 exhibits the highest score. M7, M22, M29, and M30 all exhibit high scores in the case of splice-site case mutation rates. We also consider mutations only affecting the known 99 ASD genes[CITATION] and find that, in the case of \textit{de novo} LoF SNV rates, M24 exhibits the highest score, and in the case of \textit{de novo} splice-site SNV rates, M17 shows the highest score. It should be noted that the transcripts affected by these splice-site SNVs in M17 are all derived from TCF7L2, which is a key player in canonical Wnt signalling in addition to being a high-confidence ASD gene [CITATION]. 

\section{Materials and Methods}
All analyses were performed using R version $\geq$ 3.5.1. False discovery rate (FDR) adjustment was used to correct for multiple hypothesis testing with a significance threshold of 0.05.

\subsection{Pre-processing of RNA-Seq data}
We downloaded RNA-Seq quantification data from the BrainSpan Atlas of the Developing Human Brian [CITATION]. This resource consists of both gene-level and isoform-level counts and TPM matrices, with samples derived from post-mortem brain tissue from 57 donors aged between 8 weeks post-conception through 40 years, across a number of different brain regions, for a total of 606 initial samples. These matrices were filtered by applying a filter of TPM $\geq$ 0.1 in at least 25\% of samples in both data sets; we further restricted the data to only include genes with at least one retained isoform per the isoform-level filter and vice-versa, resulting in a total of 100,834 retained isoforms.

\subsection{Normalization and differential expression analysis}
To normalize the isoform counts data for between-sample comparability, we first performed surrogate variable analysis to detect latent batch effects [CITATION], relying on evidence from a combination of principal components analysis, relative log expression and p-value distribution visualizations to determine the number of surrogate variables that minimizes latent batch effects while avoiding the problem of overfitting (see figure []). Here, we proposed to use 11 surrogate variables for downstream analysis\par

Differential expression analysis of normalized isoform counts data was performed using the \textit{limma} R package. At its core, \textit{limma} performs differential expression analysis by fitting a linear model to each isoform expression vector. However, since simply fitting a linear model generally produces low-powered results, \textit{limma} leverages the highly-parallel nature of genomic data to borrow and incorporate strength from every isoform linear model. Further, it is highly flexible and able to model for many contrasts at once as one whole integrated experiment. These linear models are then processed using parametric empirical Bayes, which, given the parallel nature of these isoform models, incorporates global and local expression variabilities, thereby increasing the overall degrees of freedom for the estimation of isoform-wise variance. Further, \textit{limma} is also able to account for nested experiment designs, similar to fitting a linear mixed effects model, through the \textit{duplicateCorrelation} function. The BrainSpan data is designed with multiple region measurements per individual, such that there are multiple expression measurements per individual "block." The \textit{duplicateCorrelation} function of \textit{limma} is used to calculate the consensus correlation, with the constraint that all isoforms share the same intrablock consensus correlation, which is then incorporated into the linear model to account for this nested data structure. 

\subsection{Weighted co-expression network analysis}
Co-expression networks were constructed using the \textit{WGCNA} R package. This network construction package operates under the scale-free topology criterion, such that the given network has a degree distribution which follows a power law, and calculates pairwise correlations among the isoform expression data. We first transformed the counts data by adjusting the counts values using information from the surrogate variable analysis and incorporating any relevant covariates [CITE LINEAR MODELS]. This transformed counts matrix was then tested for correlation with the scale-free topology, and the network was constructed blockwise using three blocks and the power estimate result from the scale-free topology correlation calculations. 

\subsection{\textit{de novo} splice site mutations}
Variant calling was performedd on \textit{de novo} mutations from [	HOW??] from both case and control autism cohorts from the Simons Simplex Consortium (SSC) and REACH [EXPAND ACRONYM]. These datasets include [N] families with autistic children.

\section{Discussion}
